src/targets/gpu/include/migraphx/gpu/less.hpp

```diff
@@ -0,0 +1,21 @@
+
+#ifndef MIGRAPHX_GUARD_RTGLIB_DEVICE_LESS_HPP
+#define MIGRAPHX_GUARD_RTGLIB_DEVICE_LESS_HPP
+
+#include <migraphx/argument.hpp>
+#include <migraphx/config.hpp>
+#include <hip/hip_runtime_api.h>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace gpu {
+namespace device {
+
+void less(hipStream_t stream, const argument& result, const argument& arg1, const argument& arg2);
```

Comment:
This header should go under  src/targets/gpu/include/migraphx/gpu/device/less.hpp. You also need to add a  src/targets/gpu/include/migraphx/gpu/less.hpp header to create the gpu operator:

```cpp
struct hip_less : binary_device<hip_equal, device::less>
{
};
```

Same thing applies to greater.